recipes/log4cxx/recipe/meta.yaml

```diff
@@ -0,0 +1,69 @@
+#--------------------------------------------------------------------------------
+# MetTools - A Collection of Software for Meteorology and Remote Sensing
+# Copyright (C) 2018  EUMETSAT
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#--------------------------------------------------------------------------------
+
+package:
+  name: log4cxx
+  version: 0.10.0
+
+source:
+  url: http://archive.apache.org/dist/logging/log4cxx/0.10.0/apache-log4cxx-0.10.0.tar.gz
+  fn:  apache-log4cxx-0.10.0.tar.gz
+  sha256: 0de0396220a9566a580166e66b39674cb40efd2176f52ad2c65486c99c920c8c
+  patches:
+    - 001-fix-compilation.patch
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+  host:
+    - expat 2.2.*
+    - apr 1.6.*
```

Comment:
Will you have more dependencies with `apr`? Then we should add it to `conda-forge-pinning`.

Reply:
I am not sure I completely understand, sorry. 
This is the only library right now that I have which depends on apr.

Reply:
Basically the idea is documented in https://conda-forge.org/docs/maintainer/pinning_deps.html but a TL;DR: We try to keep package pins inside https://github.com/conda-forge/conda-forge-pinning-feedstock so that all packages are globally pinned to the same version and write migrators in https://github.com/regro/cf-scripts to move our package tree to newer pins. This helps us that the latest versions of all packages can be used together. Pinning something in `run` is fine but when we expect several packages to have the same pin, then the handling with the global pinnings file and the migrator bot is much easier.